src/containers/ScanBoxDetails.js

```diff
@@ -12,15 +12,17 @@ import Chart from '../components/Chart';
 import { downloadCSV } from '../utils';
 import config from '../config';
 
-const getMedian = (moneyInEther, currencyPrice = 1) => {
+export const getMedian = (moneyInEther, currencyPrice = 1) => {
```

Comment:
addition is distributive so `Median(X*rate) == Median(X)*rate`. you do not have to multiply inside by currency rate. you can do it on the result. Please remove `currencyPrice` and just convert the result